offsite-activity-signup/src/Code.js

```diff
@@ -0,0 +1,458 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+var NUM_ITEMS_TO_RANK = 5;
+var ACTIVITIES_PER_PERSON = 2;
+var NUM_TEST_USERS = 150;
+
+/**
+ * Add custom menu items when opening the sheet.
+ */
+function onOpen() {
+  var menu = SpreadsheetApp.getUi().createMenu('Activities');
```

Comment:
Optional: Most scripts end up doing this as one long continuation, rather than use a variable.